### PR TITLE
Update popper.js URL in docs

### PR DESCRIPTION
**Pull request by vsn4ik**

### CODE REVIEW

**Comment by Johann-S** [comment]
Sorry @vsn4ik but what's interest do you see to change that URL ?

File: docs/4.0/getting-started/webpack.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ import 'bootstrap/js/dist/dropdown';
 ...
 {% endhighlight %}
 
-Bootstrap is dependent on [jQuery](https://jquery.com/) and [Popper](https://popper.js.org/), so npm will install them for you if needed. But they must be explicitly provided by webpack. Add the following code to the `plugins` section in your webpack config file:
+Bootstrap is dependent on [jQuery](https://jquery.com/) and [Popper](https://popper.js.org), so npm will install them for you if needed. But they must be explicitly provided by webpack. Add the following code to the `plugins` section in your webpack config file:
```
Comment (XhmikosR) [inline docs/4.0/getting-started/webpack.md]: Plus there's no gain to remove the trailing slash which personally I always want to use.

Comment (vsn4ik) [reply docs/4.0/getting-started/webpack.md]: I do not see any good reasons to add an extra character.

Comment (XhmikosR) [reply docs/4.0/getting-started/webpack.md]: There are many good reasons. Most of them do not apply here, but still there's no real gain to save that extra character either.

In fact I've been meaning to go through all docs and fix this consistently.

**Comment by vsn4ik** [comment]
@Johann-S For unification.

Already exists `[Popper](https://popper.js.org)` on:
https://github.com/twbs/bootstrap/blame/v4-dev/docs/4.0/components/dropdowns.md#L13
https://github.com/twbs/bootstrap/blame/v4-dev/docs/4.0/components/popovers.md#L14
https://github.com/twbs/bootstrap/blame/v4-dev/docs/4.0/components/tooltips.md#L13
https://github.com/twbs/bootstrap/blame/v4-dev/docs/4.0/getting-started/webpack.md#L29

**Comment by XhmikosR** [comment]
Thanks for the PR; I addressed this in a new branch along with other similar changes.

**Pull request closed without merging**